src/mappings/GenericScheme/mapping.ts

```diff
@@ -47,6 +48,9 @@ export function handleProposalExecuted(
   if (ent != null) {
     ent.executed = true;
     ent.returnValue = event.params._genericCallReturnValue;
+    if (BigInt.fromI32(0).lt(ent.value)) {
```

Comment:
should it check if the call succeed?

Reply:
Well if the call fails the event won't happen in the first place. So I don't think we need to/ can do that.

Reply:
not sure about that as if https://github.com/daostack/arc/blob/master/contracts/controller/Avatar.sol#L58 revert the event will still be emitted

Reply:
No, we check here before calling the Executed event that the call succeeded https://github.com/daostack/arc/blob/1d95202c3d2c2daef7c8dffcf7935e236c15c24a/contracts/schemes/GenericScheme.sol#L114

Reply:
I think this should be handle at the Avatar module listen to the handle GenericCall event.
not in the genericScheme

Reply:
Ok, I changed it